src/utils/protobuf.js

```diff
@@ -42,12 +42,11 @@ module.exports = `
         repeated DetailedReview detailed_review = 7;
         required PoPR popr = 8;
         required uint32 chlu_version = 9;
-        required string orbitDb = 10;
-        optional string last_reviewrecord_multihash = 11;
-        required string hash = 12;
-        required string signature = 13;
-        required string key_location = 14;
-        optional string previous_version_multihash = 15;
+        optional string last_reviewrecord_multihash = 10;
```

Comment:
Just to be sure, you don't have to, and probably shouldn't move the numbers up. Instead you can just drop the number 10. This will allow the protobuf to parse old data without breaking things.

Reply:
I see, that makes a lot of sense. Sorry about this